templates/home.html

```diff
@@ -44,12 +44,12 @@ <h2 class="how-title">HOW IT WORKS FOR FANS</h2>
         </div>
         <p>100% of your investment goes directly to the artist</p>
         <p>You can earn money by supporting artists you believe in</p>
-        <h3>BUY SHARES IN THAT ARTISTS PROJECT</h3>
+        <h3>BUY SHARES IN AN ARTISTS PROJECT</h3>
```

Comment:
I agree with this change, but it just occurred to me: should this be "AN ARTIST'S PROJECT"? (note the apostrophe).